netmap/filter.go

```diff
@@ -69,7 +70,7 @@ func (c *Context) processFilter(f *Filter, top bool) error {
 				return fmt.Errorf("%w: '%s'", ErrInvalidNumber, f.Value())
 			}
 
-			c.numCache[f] = n
+			c.numCache[f.id()] = n
```

Comment:
We van cache `f.Value()` instead, also helps to reuse parsed values between filters.

Reply:
Won't be there collision of filters with different key/op but same value?

Reply:
Or you mean `c.numCache[f.id()] = f.Value()` ?

Reply:
There can be a collision, but it is ok as cache is purely for `string -> int` conversions, irregardless of the operation performed. So `attr <= numCache[f1.Value()]` and `attr > numCache[f2.Value()]` would work as expected even if `f1.Value() == f2.Value()` and same cached value is used.